OpenRA.Mods.Common/SpriteLoaders/PngSheetLoader.cs

```diff
@@ -141,6 +146,10 @@ void RegionsFromSlices(Png png, out List<Rectangle> regions, out List<float2> of
 
 			var framesPerRow = png.Width / frameSize.Width;
 
+			var rows = (frameAmount + framesPerRow - 1) / framesPerRow;
```

Comment:
Can you explain the calculation here? `frameAmount / framesPerRow` should already give us the amount of rows?

Reply:
I kept it from the suggestion at: https://github.com/OpenRA/OpenRA/pull/17588#discussion_r382936683

Reply:
Ah I see, so it's to guard against rounding in integer maths. Thanks.